resource_templates/ekscluster.tf

```diff
@@ -141,6 +141,10 @@ resource "tanzu-mission-control_ekscluster" "tf_eks_cluster" {
           key    = "<taint-key>"
           value  = "<taint-value>"
         }
+
+        // This field is only used for cluster UPDATE, to update the ami release version.
+        // Do not use this field for cluster CREATE
+        release_version = "<ami_release_version>"
```

Comment:
In case of this version is specified as part of the CREATE then provider thrown an error? Is this check is handled in the implementation

Reply:
Yes, it's handled over [here](https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/8f3d0bc65d8fc1a61f53824f0ef029b97890157a/internal/resources/ekscluster/resource_nodepool.go#L916-L918)